tests/integration/test_charm.py

```diff
@@ -4,26 +4,83 @@
 
 """Test glauth charm."""
 
+import asyncio
+
 import pytest
+from helpers import get_glauth_res
 from pytest_operator.plugin import OpsTest
 
-GLAUTH = "glauth"
 SERIES = ["jammy"]
```

Comment:
Since we are in the process of transitioning from Juju 2.x to 3.x, we should change the terminology from series to base. I believe series is being deprecated:

```suggestion
BASES = ["ubuntu@22.04"]
```

Just note that you will also need to update the pytest parameter and your `ops_test.deploy(...)` invocation to use base instead of series.